Update cms.rst (#604)
FirelyTeam/firely-docs-firely-server: compliance/cms.rst

.. image:: ../images/CMS-0057-ProviderDirectoryAPI.svg

To implement a |PRODIR| it is necessary to:
  #. Deploy an instance of Firely Server without security measures.
  #. Produce FHIR resources compliant with the `Da Vinci PDex Plan-Net IG Version STU 1.1.0 <https://hl7.org/fhir/us/davinci-pdex-plan-net/STU1.1>`_ Implementation Guide for storage in the unsecured Firely Server instance.
  
    * Extract resources from an existing Firely Server instance and/or generate resources from other payer systems.
    * Execute this resource generation process routinely and ensure updates are completed within 30 days of any modifications to the source data.

|PRTOPR|
-------------------

> Impacted payers (see `CMS definition <https://www.cms.gov/priorities/key-initiatives/burden-reduction/interoperability/faqs>`_) are required to provide a Payor-To-Payor API in order to enable care continuity and support patient-centered data sharing with other payors. The goal of the API is to provide the possibility of transfering relevant data from an old payor to a new payor in order to support better decision-making and avoid delays in care. The following information should be provided via Payor-To-Payor API using the corresponding implementation guides:

Reviewer: ```suggestion
Impacted payers (see `CMS definition <https://www.cms.gov/priorities/key-initiatives/burden-reduction/interoperability/faqs>`_) are required to provide a Payor-To-Payor API to enable care continuity and patient-centered data sharing. The API facilitates the transfer of relevant data from an old payer to a new payer, supporting better decision-making and avoiding delays in care. The following information should be provided via Payor-To-Payor API using the corresponding implementation guides:
```

* Claim details and encounters, excluding provider remittances and enrollee cost-sharing information (see :ref:`carin_ig`)
* Clinical data incl. laboratory data (see :ref:`us_core_ig` and Da Vinci Payer Data Exchange)
* Prior Authorization Decisions, excluding those for drugs and those that were denied (Da Vinci Payer Data Exchange)

To implement a Payor-To-Payor API it is necessary to:
  #. Enable SMART on FHIR and point Firely Server to an authorization server managing organizational access for external payors. See :ref:`feature_accesscontrol`.
  #. Expose the Patient record with all its USCDI, CPCDS, and prior authorization data elements

>   #. To update the Payor-To-Payor API data within one business day of its availability. Its recommended to either :ref:`PubSub` or :ref:`tool_fsi` to ingest the resource updates.

Reviewer: ```suggestion
  #. Update the Payor-To-Payor API data within one business day of its availability. It's recommended to either :ref:`PubSub` or :ref:`tool_fsi` to ingest the resource updates.
```

|PRAUTH|
-------------------
Impacted payers (see `CMS definition <https://www.cms.gov/priorities/key-initiatives/burden-reduction/interoperability/faqs>`_) must provide a Prior Authorization API to allow patients and providers to access key details regarding prior authorization decisions:

  #. Identify which medical items and services are covered by the payer and determine if prior authorization is required.
  #. Review the necessary information and documentation needed to submit a prior authorization request.

Additionally, payers are required to support the submission of Prior Authorization requests from providers entirely through the API and facilitate the exchange of all Prior Authorization decisions via the API.

To implement a Prior Authorization API it is necessary to:
  #. Collect a computable representation of the details regarding which medical items and services are covered by a plan and whether prior authorization is required for them. Firely is recommending to store this information via `Clinical Quality Language (CQL) <https://fire.ly/cql/>`_.

>   #. For any covered medical item and service that requires Prior Authorization, additional create a computable overview of all documentation requirements that is necessary to decide on the Prior Authorization. Information that can not be derived from the Patients EHR/EMR record may be collected via `FHIR Structured Data Capture <https://hl7.org/fhir/uv/sdc/>`_. Note that the CMS final rule does not specify any requirements around the representation of the documentation requirements.

Reviewer: ```suggestion
  #. For any covered medical item or service requiring Prior Authorization, additionally create a computable overview of all documentation requirements necessary to support the decision.. Information that can not be derived from the Patients EHR/EMR record may be collected via `FHIR Structured Data Capture <https://hl7.org/fhir/uv/sdc/>`_. Note that the CMS final rule does not specify any requirements around the representation of the documentation requirements.
```
